src/styles/main.scss

```
@@ -0,0 +1,54 @@
+$watch-size: 350px;
+$center-size: 0.06 * $watch-size;
+$first-color: rgba(255, 217, 0, 0.7);
+$second-color: rgba(255, 165, 0, 1);
+
+@mixin watch-base($watch-size) {
```

міксіни можна перевикористовувати і кожному розробнику повинно хватати назви що б зрозуміти що він робить
```suggestion
@mixin circle($size) {
```